Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -854,4 +882,192 @@ std::vector<MOL_SPTR_VECT> run_Reactants(const ChemicalReaction &rxn,
   return productMols;
 }  // end of ChemicalReaction::runReactants()
 
+// Generate a product based on a SINGLE reactant
```

Comment:
Both smarts and the mdl format allow multiple reactions and products.  It actually makes sense with some reactions.

For smarts, it uses the same form for multiple products and reactants by using dot separated  strings on either side of ">>" as in this fake reaction below:

```
[C:1][N:2].[C:3]>>[C:1][N:2].[C:3]
```

Notice the "." separating the reactants and products.